utils/install.sh

```diff
@@ -8,7 +8,7 @@ die() {
 }
 
 ## stop if apash is already installed
-[[ -d "$HOME/.apash" ]] && die "basher is already installed on [$HOME/.apash]"
+[[ -d "$HOME/.apash" ]] && die "apash is already installed on [$HOME/.apash]"
```

Comment:
Ok for this one, but there is another occurence at line 39

Reply:
Done, thanks for your great review!